scripts/config/org/icepear/echarts/origin/util/option-encode.json

```diff
@@ -3,6 +3,12 @@
   "type": "interface",
   "extends": ["OptionEncodeVisualDimensions"],
   "fields": [
+    { "name": "x", "types": ["Number", "String", "Number[]", "String[]"] },
```

Comment:
Is this what you currently used to generate the file or just for maintainability purposes and you added those fields manually?

Reply:
Yes. Add these fields in config files, run `python3 config-to-java.py` in scripts folder, copy generated files into src folder, format them, commit, push.